frontend/src/components/DatasetHistoryListItem/DatasetHistoryListItem.tsx

```diff
@@ -33,7 +33,7 @@ export const DatasetHistoryListItem: FunctionComponent<ComponentProps> = (props)
           console.log(err.message);
         });
     }
-  }, [fetching]);
+  }, [fetching, item]);
   useEffect(() => setItem(props.item), [props.item]);
```

Comment:
Instead of watching `item` on line 36 (wouldn't be useful if `fetching` is false), why not add `setFetching(true)` to the effect on line 37? Let me know if this change gives you the desired result in your tests.

Reply:
The change gives me the desired results !